arm-apimanagement/2016-10-10/swagger/apimanagement-all.json

```diff
@@ -0,0 +1,7890 @@
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "The user was successfully added to the group."
+          },
+          "204": {
+            "description": "The specified user is already a member of the specified group."
+          },
+          "405": {
+            "description": "Attempt was made to add a user to a built-in group. Built-in group membership is managed by the system.",
+            "schema": {
+              "$ref": "#/definitions/ErrorBodyContract"              
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorBodyContract"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "GroupUsers"
+        ],
+        "operationId": "GroupUsers_Delete",
+        "description": "Remove existing user from existing group.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ServiceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/GroupIdParameter"
+          },
+          {
+            "$ref": "#/parameters/UserIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+           "204": {
+            "description": "The user was successfully removed from the group."
+          },
+          "405": {
+            "description": "Attempt was made to delete a user from a built-in group. Built-in group membership is managed by the API Management service.",
+            "schema": {
+              "$ref": "#/definitions/ErrorBodyContract"
+              
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorBodyContract"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/certificates": {
+      "get": {
+        "tags": [
+          "Certificates"
+        ],
+        "operationId": "Certificates_ListByService",
+        "description": "Lists a collection of all certificates in the specified service instance.",
+        "externalDocs": {
+          "url": "https://msdn.microsoft.com/en-us/library/azure/dn783483.aspx"
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ServiceNameParameter"
+          },
+ 
```

Comment:
This violates Resource Provider Contract (RPC). @solankisamir - Were your APIs reviewed and approved by the ARM team/Azure API review board.